### DISCUSSION TITLE
期望增加"定位"功能

### DISCUSSION THREAD

**Discussion by FrontGnat437168** [xfangfang/wiliwili#452] [Ideas]
像“合集”一样，“投稿”能够定位当前视频，并且不需要手动选择。谢谢了
![PixPin_2024-12-14_14-11-49](https://github.com/user-attachments/assets/b406dcf5-9b33-40b4-810f-9b52f40224a3)
![PixPin_2024-12-14_14-12-37](https://github.com/user-attachments/assets/00283224-d507-4d37-a869-ffe7b9a0a91e)
![PixPin_2024-12-14_14-13-16](https://github.com/user-attachments/assets/1a9bebf7-3f8f-4597-b751-b3d91864abbf)
![PixPin_2024-12-14_14-14-14](https://github.com/user-attachments/assets/8edd4bc3-7566-423c-9b98-3e5abc07497b)
![PixPin_2024-12-14_14-15-23](https://github.com/user-attachments/assets/2355d3f5-317e-4b7e-bb4c-0c6fcfba3c77)

**Comment by xfangfang** [comment]
已经有对应的issue: https://github.com/xfangfang/wiliwili/issues/417

**Reply by FrontGnat437168** [reply]
抱歉啊，没有提前搜索/_ \

**Reply by xfangfang** [reply]
> 抱歉啊，没有提前搜索/_ \

客气啦，这个功能描述起来比较繁琐，可能搜索了也搜不到 ：）
